src/EditorFeatures/CSharpTest/UseIndexOrRangeOperator/UseIndexOperatorTests.cs

```diff
@@ -471,6 +471,32 @@ void Goo(string[] s)
     {
         var v1 = s[^2][^1];
     }
+}", parseOptions: s_parseOptions);
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsUseIndexOperator)]
+        public async Task TestSimple_NoIndexIndexer_SupportsIntIndexer()
```

Comment:
💡 Add a test for a type that has an `int` indexer, but does not have a `Count` or `Length` property

Reply:
sure.